### ISSUE TITLE
Proper Damage scaling on the Bronze Dragon in Oculus

### ISSUE THREAD

**Issue by Vidor94** [dalaranwow/dalaran-wow#5429]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
The Broze Dragons Shock Lanze is supposed to d a little bit of normal damage plus additional dmg based on how many debuff stacks are on the target. This Damage should be increase by 100% for every Nr. 3 spell of other Bronze Dragons. 

**Expected behaviour**: Tell us what should happen instead
Math Example:
Base Dmg: 4k
4 Debuffs for 5k each: 4*5k=20k
3 times the dmg increase: (20k +4k)*3= 72k
The Dmg done is not even close to the proper amount. What i think is the Problem (since there seems to be at least a part that scales) is that the Dmg from the debuff stacks does not get increase by the beam. basicly making the Bronze Dragon dmg output significantly worse
This would change the Math to: 
3 times the dmg increase: 20k +4k*3= 32k
This is really easy to spot since with 10 stacks and 3 beams the dmg should be around 200k but it's only around 60k.

**Steps to reproduce the problem**:

1.  Get Debuffs stacks on the Endboss of Oculus.
2. Have Friends use the Nr3. Ability.
3. Use Shocklance (Nr.1)

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
http://www.wowhead.com/the-oculus
http://www.wowhead.com/item=37859/amber-essence
http://www.wowhead.com/?spell=49840
http://www.wowhead.com/?spell=49592

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Lothloryen** [comment]
Hello there,

Thank you for the report, this issue will be fixed with the next scheduled maintenance.

**Comment by Lothloryen** [comment]
On live server already.
